src/store/modules/match/selectors.ts

```diff
@@ -1,8 +1,20 @@
+import {createSelector} from '@reduxjs/toolkit';
 import {MatchStatus} from 'types/enums/match';
 import {RootState} from '../../store';
 
 export const matchData = (state: RootState) => state.match;
 
+export const historyMatchesData2 = createSelector([matchData], match => {
+  return match.matches.filter(item => {
+    return item.status !== MatchStatus.toBePlayed;
+  });
+});
+
+export const currentMatch = createSelector(
+  [matchData],
+  match => match.currentMatch,
+);
```

Comment:
This is a simple selector, why using createSelector?
```suggestion
export const currentMatch = (state: RootState) => state.match.currentMatch
```